+nosnoc/+model/Heaviside.m

```diff
@@ -0,0 +1,46 @@
+classdef Heaviside < nosnoc.model.Base
+    properties
+        f_x
+
+        c
```

Comment:
this already exists in base, why here again?

Reply:
I don't believe it is in `nosnoc.model.Base`. As a matter of fact matlab will loudly complain if you double define a property.